### ISSUE TITLE
Add h2 to section headers in wordpress blocks

### ISSUE THREAD

**Issue by sandykadam** [visual-framework/vf-wp#1200]
Hi @kasprzyk-sz 

As we have fixed h2 for all section headers in VF, there are still some components in WP blocks where h2 is missing, please find below screenshot of microsite. H2 is missing for jobs & news blocks, like this there might be more blocks to fix this.

![Screenshot 2023-01-22 at 14 27 32](https://user-images.githubusercontent.com/1227241/213921104-2ec3694a-39bb-4ab6-b301-bc04d4800ce9.png)

EBI homepage about embl block - 

![Screenshot 2023-01-22 at 14 40 04](https://user-images.githubusercontent.com/1227241/213921662-65266056-03a3-4116-8f5f-ea42c4829f4f.png)

**Comment by nikiforosk** [comment]
The "About EMBL" section header on [the EBI home page](https://www.ebi.ac.uk/) and "Explore our research groups" on [the EBI Research page](https://www.ebi.ac.uk/research) are now h2 so I'm closing this issue.
